### ISSUE TITLE
asciiToShortname doesn't work

### ISSUE THREAD

**Issue by MartkCz** [joypixels/emoji-toolkit#6]
Version: 5.0.5
PHP version: 7.3.9

Code:
```php
$client = new \JoyPixels\Client($ruleset = new \JoyPixels\Ruleset());

$client->asciiToShortname('foo :) bar');
```

PHP Warning:  array_flip(): Can only flip STRING and INTEGER values!

in https://github.com/joypixels/emoji-toolkit/blob/master/lib/php/src/Client.php#L333

**Comment by caseyahenson** [comment]
Fix on the way!

**Comment by caseyahenson** [comment]
Resolved in release 5.5.0.
